ibm/mas_devops/roles/cp4d_wds/tasks/embeddedwds.yml

```diff
@@ -0,0 +1,310 @@
+---
```

Comment:
For all the task names can you add "embeddedwds" as per the task naming guidelines https://github.com/ibm-mas/ansible-devops/blob/master/CONTRIBUTING.md#task-naming

Reply:
updated.